diagnostics-app/src/common/config.ts

```diff
@@ -74,3 +85,25 @@ export const VISIBLE_DIAGNOSTICS_CARDS: RoutineType[] = [
   RoutineType.smartctl_check_with_percentage_used,
   RoutineType.ufs_lifetime,
 ];
+
+// Routine V2 types must have an associated argument for supportability check
+// and initialization.
+export const FanRoutineArgument: RunFanRoutineArguments = {};
+export const MemoryRoutineArgument: RunMemoryRoutineArguments = {
+  maxTestingMemKib: 10000,
+};
+/* eslint-disable camelcase*/
```

Comment:
This is a problem with the current API.

When we released the API initially, we did not make sure the namings are consistent with TS/JS naming convention. Since it is released, it is now too late to change it.

Hence I've disabled the linter here since it is referencing telemetry API types.

See b/315081026 for more information